lab-justin/gulpfile.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const mocha = require('gulp-mocha');
+const nodemon = require('gulp-nodemon');
+
+const paths = ['*.js', 'lib/*.js', 'model/*.js', 'test/*.js', 'route/*.js'];
+
+gulp.task('lint', function(){
+  gulp.src(paths)
+  .pipe(eslint())
+  .pipe(eslint.format());
+});
+
+gulp.task('test', function(){
```

Comment:
This doesn't do anything--make sure to keep your gulpfile up to date with your actual code.